app/models/chargeback.rb

```diff
@@ -294,13 +294,10 @@ def self.set_chargeback_report_options(rpt, group_by, header, tz)
       rpt.col_order = static_cols + ["display_range"]
       rpt.sortby = static_cols + ["start_date"]
     end
-    rpt.col_order.each do |c|
-      if c == tag_col
-        rpt.headers.push(Dictionary.gettext(header, :type => :column, :notfound => :titleize)) if header
-      else
-        rpt.headers.push(Dictionary.gettext(c, :type => :column, :notfound => :titleize))
-      end
 
+    rpt.col_order.each do |c|
+      header_column = (c == report_tag_field && header_for_tag) ? header_for_tag : c
+      rpt.headers.push(Dictionary.gettext(header_column, :type => :column, :notfound => :titleize))
```

Comment:
@isimluk yes, good remark, basically this situation is happening only when you don't have selected Group by Tag option (header_for_tag == nil) in UI and you need to display validation message. The `header_for_tag == nil` was [previously here](https://github.com/ManageIQ/manageiq/pull/11644/commits/4f0fb06d53b5615930b867bfe30a23cc45d16e29) to do not pass `nil` into `Dictionary.gettext(header_column, :type => :column, :notfound => :titleize)` because it was throwing error and then thanks of that validation message has not been displayed.